lib/uploadcare/entity/video_converter.rb

```diff
@@ -6,17 +6,18 @@ module Entity
     # @see https://uploadcare.com/api-refs/rest-api/v0.6.0/#operation/videoConvert
     class VideoConverter < Entity
```

Comment:
The classes `DocumentConverter` and `VideoConverter` are very similar, should we remove some duplication?